css/styles.css

```diff
@@ -0,0 +1,119 @@
+/* ====================== Global Styles  ========================== */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  line-height: 1.5;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1em;
+}
+
```

Comment:
- [ ] The background color should be white as per the Figma design
your preject:
![port1](https://user-images.githubusercontent.com/53347966/227207860-b3b164c4-a260-41da-bc8a-56f0dc80a04f.JPG)